docs/gettingstarted/quickstart.ipynb

```diff
@@ -116,13 +947,281 @@
+      "text/plain": [
+       "<Figure size 500x400 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "image/png": {
+       "height": 408,
+       "width": 513
+      }
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(nrows=1, ncols=1, figsize=(5,4))\n",
+    "plt.scatter(result[\"dt\"], result[\"sf2\"], s=20, alpha=1, color=\"#353935\")\n",
+    "plt.yscale('log')\n",
+    "plt.ylabel(\"Log (SF2)\")\n",
+    "plt.xlabel(\"Time Lag (days)\")\n",
```

Comment:
The x axis looks a bit lacking here. It would be nice to have more values than just 1000. Maybe turning minor ticks on would help?

<img width="691" alt="Screenshot 2024-03-15 at 2 05 19 PM" src="https://github.com/lincc-frameworks/tape/assets/22227694/f96b7af5-ea9c-4ce2-b61d-ab5e46126036">

https://tape--403.org.readthedocs.build/en/403/gettingstarted/quickstart.html

Reply:
Adjusted x-range